Userland/Libraries/LibWeb/HTML/NavigatorBeacon.cpp

```diff
@@ -50,7 +51,8 @@ WebIDL::ExceptionOr<bool> NavigatorBeaconMixin::send_beacon(String const& url, O
         auto& content_type = body_with_type.type;
 
         // 6.2 If the amount of data that can be queued to be sent by keepalive enabled requests is exceeded by the size of transmittedData (as defined in HTTP-network-or-cache fetch), set the return value to false and terminate these steps.
-        // FIXME: We don't have a size limit in Fetching::fetch
+        if (transmitted_data->length().has_value() && transmitted_data->length().value() > 65536)
```

Comment:
It would be great if you could assign this maximum size number to a variable with a descriptive name and link to the part of the spec where it is defined (if possible).

Also, you should be able to say `64 * KiB`, which IMO is slightly clearer.

Reply:
I've added a constant.

`64 * KiB` is clearer, I've changed to that too :)